python_ta/debug/snapshot.py

```diff
@@ -84,7 +115,7 @@ def process_value(val: Any) -> int:
             if isinstance(val, (list, set, tuple)):
                 element_ids = [process_value(element) for element in val]
                 value_entry = {
-                    "type": type(val).__name__,
+                    "type": str(type(val).__name__),
```

Comment:
I'm not sure why these changes are necessary; in what cases is `__name__` not already a string?